src/main/java/org/apache/wicket/extensions/markup/html/tree/AbstractTree.java

```diff
@@ -264,9 +256,8 @@ public Object getModelObject()
 		}
 
 		@Override
-		public void renderHead(final IHeaderResponse response)
-		{
-			super.renderHead(response);
+		public void internalRenderHead(final HtmlHeaderContainer container) {
```

Comment:
Why `internalRenderHead()` is the correct thing to do ?
It is not clear from your description.